logger.Dockerfile

```diff
@@ -1,22 +0,0 @@
-# Build the inference-agent binary
-FROM golang:1.14.0 as builder
-
-# Copy in the go src
-WORKDIR /go/src/github.com/kubeflow/kfserving
-COPY go.mod  go.mod
-COPY go.sum  go.sum
-
-RUN go mod download
-
-COPY pkg/    pkg/
-COPY cmd/    cmd/
-
-# Build
-RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o agent ./cmd/agent
-
-# Copy the inference-agent into a thin image
-FROM gcr.io/distroless/static:latest
-COPY third_party/ third_party/
-WORKDIR /
-COPY --from=builder /go/src/github.com/kubeflow/kfserving/agent .
-ENTRYPOINT ["/agent"]
```

Comment:
Do not remove this yet, it is still part of the CI although it produces the same image as agent. We will take care of this.

Reply:
Got it. So we can remove it later after CI fixed.

Reply:
@naveensrinivasan 

Seems like you should not remove this as of now

Reply:
Thanks